filters/resources.txt

```diff
@@ -2310,3 +2310,34 @@ damoh-defuser.js application/javascript
     var observer = new MutationObserver(cleanVideoAsync);
     observer.observe(document.documentElement, { childList: true, subtree: true });
 })();
+
+
+twitch-videoad.js application/javascript
+(function() {
```

Comment:
I don't see a security issue here, but since this script is site-specific, should I include a check at the beginning that verifies that `document.domain === "twitch.tv"`? For that to work securely no site scripts must be allowed to run before this script, while I think that's true it's not mentioned [in the documentation](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#scriptlet-injection).